tsup.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'tsup'
+
+export default defineConfig({
+  entry: ['src/index.ts'],
+  format: ['cjs', 'esm'],
+  dts: true,
+  minify: true,
+  // We have to bundle fs-capacitor because the author is too lazy to support cjs
```

Comment:
This comment accusing the `fs-capacitor` author of being lazy is toxic. He's put a lot of work into that package for free; his decision to publish pure ESM was not a lazy one.

Reply:
My fault. Respect to that guy. I just was moving fast and wanted to get things working.